apps/studio/components/layouts/AdvisorsLayout/AdvisorsMenu.utils.ts

```diff
@@ -28,5 +32,20 @@ export const generateAdvisorsMenu = (project?: Project): ProductMenuGroup[] => {
         },
       ],
     },
+    ...(IS_PLATFORM && features?.advisorRules
+      ? [
+          {
+            title: 'Configuration',
+            items: [
+              {
+                name: 'Advisor Rules',
+                key: 'advisor-rules',
```

Comment:
nit: this menu item doesn't show as active:

<img width="280" alt="Screenshot 2025-03-25 at 15 17 12" src="https://github.com/user-attachments/assets/2b71145c-c18a-47b5-92dc-5f9d7cda64f8" />